### ISSUE TITLE
Use Environment Variables and Avoid Config Step

### ISSUE THREAD

**Issue by nelsonic** [dwyl/elixir-auth-github#35]
At present the [`README.md`](https://github.com/dwyl/elixir-auth-github/tree/44021233add025348f261f5bf0b9b32fe6c5711a) recommends using Environment Variables but it requires you to add `config` settings to your `config.exs`:
![image](https://user-images.githubusercontent.com/194400/71510888-4fefcb80-2888-11ea-9191-f4d02106db52.png)

We could avoid the step and just inform people they _must_ have **`GITHUB_CLIENT_ID`** and **`GITHUB_CLIENT_SECRET`** defined in order for the package to work.

**Comment by nelsonic** [comment]
Thinking of raising an error if the environment variables are undefined:
```
iex> raise ArgumentError, message: "Environment variables GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET undefined"
```

see: https://hexdocs.pm/elixir/Exception.html 
and: https://elixir-lang.org/getting-started/try-catch-and-rescue.html#errors

**Comment by nelsonic** [comment]
Alternatively the more succinct and _fatal_:
```elixir
iex> exit("Environment variables GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET undefined")
** (exit) "Environment variables GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET undefined"
```

I want to figure out the appropriate way of doing Logging: https://github.com/dwyl/learn-elixir/issues/149

**Comment by nelsonic** [comment]
At present the [`/test/elixir_auth_github_test.exs`](https://github.com/dwyl/elixir-auth-github/blob/44021233add025348f261f5bf0b9b32fe6c5711a/test/elixir_auth_github_test.exs) file has 28 mentions of the word "env" ...
![image](https://user-images.githubusercontent.com/194400/73590973-e1cea200-44e0-11ea-8f03-098782bdc10a.png) 
![image](https://user-images.githubusercontent.com/194400/73591250-2dcf1600-44e4-11ea-8f0d-b79357360432.png)
_Most_ of the tests include some reference to the "ENVIRONMENT VARIABLES NOT SET" ...
This feels unnecessarily repetitive ... 🤔

**Comment by nelsonic** [comment]
Resolved by #34 ✅
